File: connectors/connector-github/src/test/scala/com/taskadapter/connector/github/GithubConfigTest.scala
package com.taskadapter.connector.github

import org.junit.runner.RunWith
import org.scalatest.junit.JUnitRunner
import org.scalatest.{FunSpec, Matchers}

@RunWith(classOf[JUnitRunner])
class GithubConfigTest extends FunSpec with Matchers {

  it("defaultLabelIsSet") {
    new GithubConfig().getLabel shouldBe GithubConfig.DEFAULT_LABEL
  }
}

File: connectors/connector-github/src/test/java/com/taskadapter/connector/github/GithubConfigTest.java
package com.taskadapter.connector.github;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class GithubConfigTest {
    @Test
    public void defaultLabelIsSet() {
        assertThat(new GithubConfig().getLabel()).isEqualTo(GithubConfig.DEFAULT_LABEL);
    }
}
